recipes/openmvg/all/conanfile.py

```diff
@@ -30,13 +30,15 @@ class Openmvgconan(ConanFile):
         "with_openmp": [True, False],
         "with_avx": [False, "avx", "avx2"],
         "programs": [True, False],
+        "jpeg": ["libjpeg", "libjpeg-turbo"]
```

Comment:
Could you change this option name to be consistent with other CCI recipes please? There is already enough inconsistencies in CCI ;)

Also you can add mozjpeg as other CCI recipes already do.

Reply:
Done

Reply:
`"with_libjpeg":` - 10 recipes
`"with_jpeg":` - 21 recipes
`"jpeg":` - 5 recipes
Therefore I stopped on `with_jpeg`